rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/50_filter.yml

```diff
@@ -32,7 +32,8 @@ setup:
       indices.refresh: {}
 
 ---
-"Filter aggs with terms lookup ensure not cached":
+"Filter aggs with terms lookup and ensure it's cached":
+  # Because the filter agg rewrites the terms lookup in the rewrite phase the request can be cached
   - skip:
       version: " - 5.0.0"
```

Comment:
I think you have to skip before 6.0 because a mixed cluster may or may not cache depending on where the shards live.